### PR TITLE
Generator for dotenv

### PR DESCRIPTION
**Pull request by mausamp**

### CODE REVIEW

File: lib/generators/boring/dotenv/install/install_generator.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+module Boring
+  module Dotenv
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds dotenv gem to the application'
+
+      def add_dotenv_gem
+        say 'Adding dotenv gem', :green
+
+        Bundler.with_unbundled_env do
+          run 'bundle add dotenv-rails'
```
Comment (abhaynikam) [inline lib/generators/boring/dotenv/install/install_generator.rb:12]: can you add this in development env?

File: lib/generators/boring/dotenv/install/install_generator.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        end
+      end
+
+      def configure_dotenv_gem
+        say 'Configuring dotenv gem', :green
+
+        create_file '.env', <<~CONTENT
```
Comment (abhaynikam) [inline lib/generators/boring/dotenv/install/install_generator.rb:19]: can we use the template here instead?

**Review by abhaynikam — COMMENTED** [review]
Left some comments

**Comment by mausamp** [comment]
@abhaynikam I have resolved the comments, can you please take a look.

**Pull request merged**